dialogue/Story/Intro/IntroViolastro4.json

```diff
@@ -8,14 +8,18 @@
       "See y'all never, low lives!"
     ],
     "es": [
-      "��Hasta nunca, pringaos!!"
+      "¡¡Hasta nunca, pringaos!!"
+    ],
+    "pl": [
+      "Do zobaczenia nigdy, przegrywy!"
```

Comment:
Would it be better to use "fajerzy" instead? "Przegrywy" is a more literal meaning of the word loser, while the use of "losers/low lives" here is used as an insult, although I am not sure how offensive it is considered to be.

Reply:
I think that both "frajerzy" and "przegrywy" fit, but "frajerzy" could actaully sound a little better. Thanks for the suggestion